<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AlginmentReligion</class>
 <widget class="QWidget" name="AlginmentReligion">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>616</width>
    <height>513</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="lblTitle">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>34</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Alignment And Religion</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblAlignment">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Aligment:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblAboutReligion">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>About Selected Alignment</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblReligion">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>62</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Religion:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblAboutReligion_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>About Selected Religion</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cboxAlignment">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>60</y>
     <width>231</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="cboxReligion">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>270</y>
     <width>201</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAboutAlignment">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>60</y>
     <width>141</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>About Alignment</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAboutReligion">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>270</y>
     <width>141</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>About Religion</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textAlignment">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>591</width>
     <height>151</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textReligion">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>591</width>
     <height>141</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnNext">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>470</y>
     <width>114</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnback">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>470</y>
     <width>114</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
